### ISSUE TITLE
Issue related php8.2

### ISSUE THREAD

**Issue by hgati** [magenizr/Magenizr_Envbar#1]
Here, a deprecated error is also occurring.

![](https://gcdnb.pbrd.co/images/sRedGrHFM5pj.png)

https://github.com/magenizr/Magenizr_Envbar/blob/56d49b9dcfe1cc21c58d39fc9139c78b779529ca/Helper/Data.php#L41
https://github.com/magenizr/Magenizr_Envbar/blob/56d49b9dcfe1cc21c58d39fc9139c78b779529ca/Helper/Data.php#L42
https://github.com/magenizr/Magenizr_Envbar/blob/56d49b9dcfe1cc21c58d39fc9139c78b779529ca/Helper/Data.php#L43

https://github.com/magenizr/Magenizr_Envbar/blob/56d49b9dcfe1cc21c58d39fc9139c78b779529ca/Observer/Adminhtml/AddCss.php#L30
https://github.com/magenizr/Magenizr_Envbar/blob/56d49b9dcfe1cc21c58d39fc9139c78b779529ca/Observer/Adminhtml/AddCss.php#L31

It works fine except for the above issue.

**Comment by hgati** [comment]
I've patched to fix issue in php8.2

https://gist.github.com/hgati/8fc917aec82b4259116f87c61dd3c51b#file-0001-declare-class-property-to-fix-issue-in-php-8-2-patch

**Comment by magenizr** [comment]
Thanks @hgati 

I'll provide an update for Envbar tomorrow.

Regards,
John

**Comment by magenizr** [comment]
Hi @hgati 

Btw. when you have time - please reach out to us via hello@magenizr.com.au so that we can stay connected and eventually discuss some module tasks.

**Comment by hgati** [comment]
**Another Issue,**
After installing the 'envvar' module, the UI of the system configuration breaks in several places.

**Cron** 
![](https://gcdnb.pbrd.co/images/WKLzxNkkLj9y.png)

**Payment Method**
![](https://gcdnb.pbrd.co/images/5VKpCyttN3vc.png)

**Media Gallly Folder**
![](https://gcdnb.pbrd.co/images/4q4SnGFEE28O.png)

**Comment by hgati** [comment]
Okay, I've resolved the issue caused by incorrect CSS triggered by the AddCss function. 
(vendor/magenizr/magento2-envbar/Observer/Adminhtml/AddCss.php)
The key point is to remove the CSS related to .form-list. It seems to have been added by mistake.

https://gist.github.com/hgati/0e2ae9415f217e1c168cc0f2870b2965#file-0002-magenizr-magento2-envbar-fix-system-config-form-field-ui-broken-patch

With the patch, now the Add button and input HTML tag are displayed correctly. 
Additionally, the system config UI related to cron and payment method, as mentioned above, is also displayed correctly. 

![](https://gcdnb.pbrd.co/images/WQWThtw99Zp6.png)
